applications/klighd-vscode/src/klighd-webview.ts

```diff
@@ -49,4 +81,18 @@ export class KLighDWebview extends SprottyLspWebview {
             })
         );
     }
+
+    /** Toggles "sync with editor". If disabled, the diagram view will not update when the active editor changes. */
+    toggleSyncWithEditor(sync: boolean): void {
```

Comment:
This method name is a bit misleading: a toggle function should not have a parameter as just toggling flips the option. I guess this should rather be named `setSyncWithEditor`.

Reply:
Refactored.